### ISSUE TITLE
new naming ideas

### ISSUE THREAD

**Issue by cgwalters** [jlebon/redhat-ci#29]
"PAPR" - Project Atomic Pull Requests (currrent state)
"CCIPR" - CentOS CI PR tester

My vote here is to explicitly limit the scope to PR testing - i.e. we don't try to do delivery.

**Comment by jlebon** [comment]
Thanks for starting the thread! I'd definitely like to have it renamed before moving it under the Project Atomic org.

PAPR sounds cool, I like that it's easy to say (I think it'd still be an appropriate name even after moving to CentOS CI since a lot of the features are tailored for Project Atomic, e.g. ostree support and package layering).

Some other ideas off the top of my head:

"Atomic CI" - self-explanatory
"AUPR" (pronounced like "upper") - Atomic upstream PR

Or if we want to stay in the atomic/physics field, how about "quark"? (A nucleus is made of quarks, and we help grow the atom :)). There's some [prior](https://en.wikipedia.org/wiki/Quark_(company)) [use](https://en.wikipedia.org/wiki/Quark_(kernel)), though it should be pretty clear from context what we mean.

> My vote here is to explicitly limit the scope to PR testing - i.e. we don't try to do delivery.

Agreed. Though I still want to eventually build RPMs just for the scope of PR testing (as mentioned in https://github.com/jlebon/redhat-ci/issues/10). This will also help projects hooked in an rdgo-style CD to be even more reliable (though that might be awkward in a PR workflow, unless projects start keeping their specs in-tree).

Oh I just realized, did you say this just to mean that we should avoid "CI" in the name?

**Comment by jlebon** [comment]
I'd like to get the repo moved soon.

I think I'd prefer to name it something pronounceable/a real word. Between "papr" and "quark", I'm leaning slightly more towards the latter for the cool factor, though the former is nice too since it naturally pronounces as "paper" and stands for something. Any preferences on the options so far or with other suggestions? (This is directed not just to @cgwalters; subscribed folks, feel free to chime in as well!)

This is all a bit bikeshed of course since this is not a project that AH users will ever actually interact with, so I don't want to spend too much time on this either. Let's try to get it renamed and moved in the next week or so.

**Comment by miabbott** [comment]
My vote for the shed is PAPR

**Comment by jlebon** [comment]
OK, first step is in https://github.com/jlebon/redhat-ci/pull/41.

Once that's done, I'll rename the repo, work out any snags, and then we can start the process to move it to the projectatomic/ org.

**Comment by cgwalters** [comment]
Although...something we should probably figure out here is; is there a name for the combination of PAPR+Homu?  For people using them together, they're fairly entangled/inseparable.

Until now I've been saying "projectatomic-ci-infra" for the combination, but obviously that's an unwieldy thing to write.  We could call the *combination* PAPR?  (And hence limit our scope to PRs?)

**Comment by cgwalters** [comment]
Then this project would be papr-executor or something?

**Comment by jlebon** [comment]
Hmm, "papr-executor" also sounds a bit wordy. How about "PACI" for (PAPR+Homu)?

**Comment by cgwalters** [comment]
That's fine by me!

**Comment by jlebon** [comment]
The repo has been renamed! 🆎 ➡️ 🔤
Going to close this now.

Still in the process of adapting to the name change, will submit request to atomic-devel shortly.

**Comment by cgwalters** [comment]
Not sure whether to reopen this but...one angle to this is the "user interface".  People are often (rightly!) confused by the part that's still called "Red Hat CI Services" in https://github.com/jlebon/papr/blob/master/README.md#more-details-about-red-hat-ci-services

Basically...do we rename @rh-atomic-bot to @projectatomic-ci ?  (Or just @projectatomic)?  And more importantly, do we try to have e.g. `$botname retry` actually talk to PAPR in the case where the PR isn't queued for merge?

**Comment by cgwalters** [comment]
Another angle is: we should really prioritize testing  `auto` over PRs.  And doing things like that is going to result in greater entanglement between the two in the code.
